_rules/SC1-2-1-audio-transcript.md

```diff
@@ -21,21 +18,21 @@ authors:
 
 ### Applicability
 
-The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or with a "play button".
+The rule applies to any [non-streaming](#non-streaming-media-element) `audio` element that is playing or has a "play button". The play button must be [visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
 
-A play button is an interactive element that when activated, plays the audio. The play button must be ]visibile on the page](#visible-on-the-page) or [exposed to assistive technologies](#exposed-to-assistive-technologies)
+**Note:** A play button is an interactive element that when activated, plays the audio. 
 
 ### Expectation
 
-The auditory information of each test target is available through a caption, or through a [text transcript](#text-transcript) that is available either on the page or through a link.
+The auditory information of each test target is available through a [text transcript](#text-transcript) that is available either on the page or through a link.
 
 ## Assumptions
 
 *There are currently no assumptions*
 
 ## Accessibility support
 
-While the HTML specifications allows the use of `track` elements inside of `audio` elements, there is currently limited to no support for this in major web browsers.
+There are no major accessibility support issues known for this rule.
 
 ## Background
 
```

Comment:
Make these links more readable [Title of page](link of page)

Reply:
Fixed. Thanks.